doc/concepts/cable_cell.rst

```diff
@@ -9,15 +9,39 @@ channels, synapses, gap junction sites, stimuli and spike detectors.
 
 Cable cells are constructed from three components:
 
-* :ref:`Morphology <morph>`: a decription of the geometry and branching structure of the cell shape.
-* :ref:`Label dictionary <labels>`: a set of rules that refer to regions and locations on the cell.
+* :ref:`Morphology <morph>`: a description of the geometry and branching structure of the cell shape.
+* :ref:`Label dictionary <labels>`: a set of definitions and a :abbr:`DSL (domain specific language)` that refer to regions and locations on the cell morphology.
 * :ref:`Decor <cablecell-decoration>`: a description of the dynamics on the cell, placed according to the named rules in the dictionary.
 
-When a cable cell is constructued the following steps are performed using the inputs:
+The decor can make reference to particular mechanisms and mechanism catalogues. More on mechanisms at :ref:`mechanisms`.
```

Comment:
Maybe this can be shortened and added to the bullet point of the decor? 
:ref:`Decor <cablecell-decoration>`: a description of the dynamics on the cell, placed according to the named rules in the dictionary. It can reference :ref:`mechanisms from mechanism catalogues. <mechanisms>